File: 프로그래머스/lv0/120887. k의 개수/k의 개수.js
function solution(i, j, k) {
  var answer = 0;
  for (let index = i; index < j+1; index++) {
    const numberToStr = String(index);
    if (numberToStr.includes(String(k))) {
      for (const checkItem of numberToStr) {
        if (checkItem === String(k)) {
          answer += 1;
        }
      }
    }
  }
  return answer;
}

File: 프로그래머스/lv0/120887. k의 개수/k의 개수.py
def solution(i, j, k):
    answer = 0
    k = str(k)
    
    for num in range(i, j + 1):
        num = str(num)
        for check in range(0, len(num)):
            if num[check] == k:
                answer += 1
    
    return answer
